Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/AuditTransformDataBuilder.java

```diff
@@ -0,0 +1,194 @@
+ *     * Neither the name of the United States Government nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.audit;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+ *
+ * @author achidamb
+ */
+public class AuditTransformDataBuilder {
+
+    public Map<String, String> eventIdCodeMap;
+    public Map<String, String> eventCodeSystemMap;
+    public Map<String, String> eventDisplayNameRequestorMap;
+    public Map<String, String> eventDisplayNameResponderMap;
+    public Map<String, String> eventTypeCodeMap;
+    public Map<String, String> eventTypeCodeSystemMap;
+    public Map<String, String> eventTypeCodeDisplayNameMap;
+    public Map<String, String> eventActionCodeRequestorMap;
+    public Map<String, String> eventActionCodeResponderMap;
+
+    private AuditTransformDataBuilder() {
+        eventIdCodeMap = createEventIdCodeMap();
+        eventCodeSystemMap = createEventCodeSystemMap();
+        eventDisplayNameRequestorMap = createEventDisplayNameRequestorMap();
+        eventDisplayNameResponderMap = createEventDisplayNameResponderMap();
+        eventTypeCodeMap = createEventTypeCodeMap();
+        eventTypeCodeSystemMap = createEventTypeCodeSystemMap();
+        eventTypeCodeDisplayNameMap = createEventTypeCodeDisplayNameMap();
+        eventActionCodeRequestorMap = createEventActionCodeRequestorMap();
+        eventActionCodeResponderMap = createEventActionCodeResponderMap();
+    }
+
+    private Map<String, String> createEventIdCodeMap() {
+        eventIdCodeMap = new HashMap();
+        eventIdCodeMap.put("PatientDiscovery", "110112");
+        return eventIdCodeMap;
+    }
+
+    /**
+     *
+     * @param serviceName
+     * @return
+     */
+    public String getEventIdCode(String serviceName) {
+        if (eventIdCodeMap.containsKey(serviceName)) {
+            return eventIdCodeMap.get(serviceName);
+        }
+        return null;
+    }
+
+    private Map<String, String> createEventCodeSystemMap() {
+        eventCodeSystemMap = new HashMap();
```

Comment:
Since we're using JDK 7 now, use the diamond operator **EVERYWHERE** you invoke "new" on a type that accepts generics, e.g.:

new HashMap<>();
new ArrayList<>();
etc.